### PR TITLE
Extended Support to OV v2800 and I3s 2020

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
PR Extends SDK Support to OV REST v2800 and I3s REST v2020

### Issues Resolved
N/A

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,45 @@
...
+- Hypervisor Cluster Profile
+- Hypervisor Manager
+- Interconnects
+- Interconnect Type
+- LIG Uplink Set
+- Logical Enclosure
+- Logical Interconnect
+- Logical Interconnect Group
+- Network Set
+- OS Deployment Plan
```
Comment (yuvirani) [inline CHANGELOG.md:27]: Add lib and spec files for os_deployment_plan as it is being used by HCP.

Comment (AsisBagga) [reply CHANGELOG.md:27]: fixed

**Review by yuvirani — APPROVED** [review]

File: lib/oneview-sdk/image-streamer/resource/api2020/deployment_plan.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# (C) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
```
Comment (nabhajit-ray) [inline lib/oneview-sdk/image-streamer/resource/api2020/deployment_plan.rb:2]: Change copyright

Comment (AsisBagga) [reply lib/oneview-sdk/image-streamer/resource/api2020/deployment_plan.rb:2]: updated

File: lib/oneview-sdk/image-streamer/resource/api2020/artifact_bundle.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# (C) Copyright 2020 Hewlett Packard Enterprise Development LP
```
[the remark this answers is not in the document]
Comment (AsisBagga) [reply lib/oneview-sdk/image-streamer/resource/api2020/artifact_bundle.rb]: updated

File: lib/oneview-sdk/image-streamer/resource/api2020.rb [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
```
Comment (nabhajit-ray) [inline lib/oneview-sdk/image-streamer/resource/api2020.rb:1]: Change here also.

**Review by nabhajit-ray — APPROVED** [review]

**Pull request merged**